src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -698,12 +698,12 @@ private void tryTriggerFloatingMedrec(final WebElement slot) {
         @Override
         public Object apply(WebDriver webDriver) {
           jsActions.execute(
-              "window.scroll(0, 5000); setTimeout(function () {window.scroll(0, 5001) }, 500)");
+              " (function(){ window.scroll(0, 5000); setTimeout(function () {window.scroll(0, 5001) }, 100); })(); ");
           return slot.getAttribute("style").contains("visibility: visible;");
         }
       });
     } catch (org.openqa.selenium.TimeoutException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)